desktop/core/src/desktop/lib/fs/ozone/upload.py

```diff
@@ -29,9 +30,110 @@
 else:
   from django.utils.translation import ugettext as _
 
+from desktop.lib.exceptions_renderable import PopupException
+from filebrowser.utils import generate_chunks
 
 LOG = logging.getLogger()
 
+class OFSFineUploaderChunkedUpload(object):
+  def __init__(self, request, *args, **kwargs):
+    self.qquuid = kwargs.get('qquuid')
+    self.qqtotalparts = kwargs.get('qqtotalparts')
+    self.totalfilesize = kwargs.get('qqtotalfilesize')
+    self.file_name = kwargs.get('qqfilename')
+    self.chunk_size = UPLOAD_CHUNK_SIZE.get()
+    self.destination = kwargs.get('dest', None)  # GET param avoids infinite looping
+    self.target_path = None
+    self.file = None
```

Comment:
What is this `file` for? Where is this used?

Reply:
kept it for same as old class. Not used in new class.